app/views/layouts/_modal_dialog.html.erb

```diff
@@ -0,0 +1,11 @@
+<aside class='dialog' id=<%= content_for :modal_id %> >
+  <h2>
+    <%= yield_content! :modal_title %>
+  </h2>
+  <br/>
+  <%= raw yield_content! :modal_content %>
```

Comment:
@b2simms I just got rid of the `raw` and both dialogs seemed to work fine for me. Can you please try again after making the other changes?